Remotion/Web/Development.WebTesting/WebTestSetUpFixtureHelper.cs

```diff
@@ -133,13 +141,50 @@ public void OnSetUp ()
     public void OnTearDown ()
     {
       UnhostWebApplication();
+      _remoteBrowserDockerContainer?.Dispose();
     }
 
     private void HostWebApplication ()
     {
       _hostingStrategy.DeployAndStartWebApplication();
     }
 
+    private void HostDriverBrowserInDocker (IWebTestSettings webTestSettings)
+    {
+      var remoteDriverSettings = webTestSettings.RemoteDriver;
+      var docker = new DockerCommandLineClient(TimeSpan.FromMinutes(3), _loggerFactory);
+
+      var imageNameWithPlaceholders = remoteDriverSettings.DockerImageName ?? "selenium/standalone-$browsername$:$browserversion$.0";
```

Comment:
Question on API consistency: wouldn't `{browsername}` better since it aligns with composite strings / named holes instead of inventing a new placeholder syntax?

Reply:
Using braces will make it harder if you use interpolated strings. I am not really invested in any kind of placeholder syntax, I just think we need a placeholder syntax here to make it more useful. Irrespective of the placeholder syntax used, you need to know that the syntax exists, and as we don't have an established syntax, we have to pay the price anyway

Reply:
yes. my reasoning is simply, if we copy the .net syntax, there's no extra cost to pay for developers as they know the syntax already and don't think about it.